src/gei/Commands/MigrateRepo/MigrateRepoCommand.cs

```diff
@@ -102,6 +103,11 @@ public MigrateRepoCommand() : base(
         {
             Description = "Only effective if migrating from GHES. Disables SSL verification when communicating with your GHES instance. All other migration steps will continue to verify SSL. If your GHES instance has a self-signed SSL certificate then setting this flag will allow data to be extracted."
         };
+        public Option<bool> UseGithubStorage { get; } = new("--use-github-storage")
```

Comment:
Same as bbs2gh one, this also needs to be added to [MigrateRepoCommandArgs](https://github.com/github/gh-gei/blob/0bfb32f806c4824475df80a4886e3d647dd3fa07/src/gei/Commands/MigrateRepo/MigrateRepoCommandArgs.cs) plus some validations.

Reply:
Its merging into a feature branch! Check out the next PR https://github.com/github/gh-gei/pull/1270